build.gradle

```diff
@@ -206,33 +231,34 @@ task yarn(dependsOn: setupYarn) {
 }
 
 task buildEnigma(type: Zip) {
+	group = buildMappingGroup
 	from mappingsDir
 	include "**/*"
 	archiveName "yarn-enigma-${yarnVersion}.zip"
 	destinationDir(file("build/libs"))
 }
 
-task checkMappings {
+task checkMappings(dependsOn: mergeJars) {
+	group = "verification"
 	inputs.dir mappingsDir
-	doLast {
-		logger.lifecycle(":checking mappings")
 
-		String[] args = [
-			mergedFile.getAbsolutePath(),
-			mappingsDir.getAbsolutePath()
-		]
+	outputs.upToDateWhen {false}
 
-		new CommandFindMappingErrors().run(args)
+	doLast {
+		logger.lifecycle(":checking mappings")
+		new CheckMappingsCommand().run mergedFile.getAbsolutePath(), mappingsDir.getAbsolutePath()
```

Comment:
Put the brackets into this, I didnt know this was possible, but keep it consisitant with the rest of the file.

Reply:
:sweat: